src/components/App.tsx

```diff
@@ -0,0 +1,23 @@
+import React, { Component } from 'react';
+import { Switch, Route, BrowserRouter as Router } from 'react-router-dom';
+import { withOidcSecure } from '@axa-fr/react-oidc-context';
+import { Home } from './index';
+import PageContainer from '../containers/PageContainer';
+
+const ProtectedHome = withOidcSecure(Home);
+
+class App extends Component {
+    render() {
+        return (
+            <Router>
+                <PageContainer>
+                    <Switch>
+                        <Route exact path="/" component={ProtectedHome} />
```

Comment:
nit: just call `withOidcSecure` here to remove the instantiation of another variable. This pattern will just end with a huge list of variables at the top of the file and make this less readable.
```suggestion
                        <Route exact path="/" component={withOidcSecure(Home)} />
```